securedrop/client.py

```diff
@@ -37,31 +40,86 @@ def make_dict(self):
         }
 
 
+class AESWrapper(object):
+
+    def __init__(self, key):
+        self.bs = AES.block_size
+        self.key = hashlib.sha256(key.encode()).digest()
+
+    def encrypt(self, raw):
+        verify = hashlib.sha256((raw.encode())).hexdigest()
+        raw = self._pad(raw)
+        iv = Random.new().read(AES.block_size)
```

Comment:
Should we use `Crypto.Random.get_random_bytes()` here instead? It seems a bit simpler than creating a new random object. Also might as well implement `make_salt` with the same function while you're at it.

Reply:
Done.